Grid.elm

```diff
@@ -52,10 +46,52 @@ addShip ship grid =
   shipCoordinates
     |> List.foldr setToShip grid
 
+cellToHtml : Cell -> Html.Html
+cellToHtml cell =
```

Comment:
So, I'm all for making things look prettier, but not yet. The main focus right now is getting the functionality working. That's a big reason why I am using the `Matrix.prettyPrint` function in the first place. It allows a really nice temp ui while we are creating the game.